services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -184,22 +186,48 @@ testCreateTwoLegalHoldDevices = do
 
 testApproveLegalHoldDevice :: TestM ()
 testApproveLegalHoldDevice = do
-    pure ()
+    (owner, tid) <- createTeam
+    member <- randomUser
+    addTeamMemberInternal tid $ newTeamMember member (rolePermissions RoleMember) Nothing
+
+    cannon <- view tsCannon
+
+    WS.bracketR cannon member $ \ws -> withDummyTestServiceForTeam owner tid $ do
+        -- not allowed to approve if team setting is disabled
+        -- TODO: remove the following 'ignore' once 'disabled' is the default
+        ignore $ approveLegalHoldDevice owner member tid !!! const 403 === statusCode
```

Comment:
can you test that this also happens if you:

> enable -> request device -> disable -> approve

Reply:
Question; should disabling cause pending devices to be deleted? I'd assume it's easiest if we don't?

Reply:
not sure.  i would have tried to delete it, but can't think of a strong reason why we should.